jenkins/check-for-build.jenkinsfile

```diff
@@ -17,6 +17,7 @@ pipeline {
             H 1 * * * %INPUT_MANIFEST=2.9.1/opensearch-2.9.1.yml;TARGET_JOB_NAME=distribution-build-opensearch;BUILD_PLATFORM=linux macos windows;BUILD_DISTRIBUTION=tar rpm deb zip
             H 1 * * * %INPUT_MANIFEST=2.10.0/opensearch-dashboards-2.10.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip
             H 1 * * * %INPUT_MANIFEST=2.10.0/opensearch-2.10.0.yml;TARGET_JOB_NAME=distribution-build-opensearch;BUILD_PLATFORM=linux macos windows;BUILD_DISTRIBUTION=tar rpm deb zip
+            H 1 * * * %INPUT_MANIFEST=2.10.0/opensearch-2.10.0.yml;TEST_MANIFEST=2.10.0/opensearch-2.10.0-concurrent-search-test.yml;TARGET_JOB_NAME=distribution-build-opensearch-concurrent-search;BUILD_PLATFORM=linux macos windows;BUILD_DISTRIBUTION=tar rpm deb zip
             H 1 * * * %INPUT_MANIFEST=2.8.1/opensearch-dashboards-2.8.1.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows;BUILD_DISTRIBUTION=tar rpm deb zip
```

Comment:
This should be as `TARGET_JOB_NAME=distribution-build-opensearch`, you can ignore the `BUILD_PLATFORM`.

Reply:
Thanks. updated

Reply:
@prudhvigodithi @sohami  This should not be updated in check for build. Cronjob for integtest should be added directly in integ-test file here: https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/integ-test.jenkinsfile

Reply:
You can ignore adding any cronjob for now. I can take it up later. Please only proceed with adding the concerned manifest.
Thanks!

Reply:
removed the jenkins file change. @gaiksaya Can you help with adding the cron related change to run the test with custom manifest file. We want to see if there are any plugin test failures with concurrent search enabled

Reply:
Will do!

Reply:
Created the cron here: https://github.com/opensearch-project/opensearch-build/pull/3857